sstables/sstables_manager.cc

```diff
@@ -61,4 +67,44 @@ sstable_writer_config sstables_manager::configure_writer() const {
     return cfg;
 }
 
+void sstables_manager::add(sstable* sst) {
+    _active.push_back(*sst);
+    smlogger.debug("activated sstable, {} active sstables", value_of([this] { return _active.size(); }));
```

Comment:
Is it easy to deduce from other logs which sstable just got activated? If not, perhaps logging its path as well would make debugging easier

Reply:
Those are really debugging aids that I hastily s/info/debug/'d in prepartion for submission and didn't consider much. I'll give them a look and see if they really need to exist, and if so what information they should provide.